src/background/navigationListener.ts

```diff
@@ -6,9 +6,9 @@ export function navigationListener(): void {
   chrome.webNavigation.onCompleted.addListener((details) => {
     if (details.frameId === 0) {
       console.log('Main frame navigation completed:', details);
-      sendPageView(details.url);
+      sendPageView();
```

Comment:
Is this correct? In which case are we sending an empty payload here?

Reply:
fixed 👍